doc/cephfs/experimental-features.rst

```diff
@@ -70,7 +68,7 @@ Note that snapshots and multiple filesystems are *not* tested in combination
 and may not work together; see above.
 
 Multiple filesystems were available starting in the Jewel release candidates
-but were protected behind the "enable_multiple" flag before the final release.
+but were protected behind the ``enable_multiple`` flag before the final release.
```

Comment:
`but must be turned on via the ... until declared stable.`

Reply:
@batrick Done. Please check again. Thanks.